src/api/app/views/webui/user/show.html.haml

```diff
@@ -0,0 +1,170 @@
+        %br/
+      - if @configuration['change_password'] && CONFIG['proxy_auth_mode'] != :on
+        = link_to(sprited_text('key', 'Change your password'), { controller: 'user', action: 'password_dialog', user: User.current }, {id: 'password_dialog', remote: true})
+        %br/
+      = link_to(sprited_text('email', 'Change your notifications'), user_notifications_path)
+      %br/
+.grid_12.omega.box.box-shadow
+  #involved
+    .box-header.header-tabs
+      %ul
+        %li
+          %a{href: "#ipackages", title: "Packages that you are involved with"} Involved Packages
+        %li
+          %a{href: "#iprojects", title: "Projects that you are involved with"} Involved Projects
+        %li
+          %a{href: "#iowned", title: "Projects/Packages where you are responsible for the maintenance"} Owned Project/Packages
+    #ipackages.tab
+      - if @ipackages.length > 0
+        #ipackages_wrapper{ "data-url": url_for(controller: 'package', action: 'show', project: 'REPLACEPRJ', package: 'REPLACEPKG') }
+          - content_for :ready_function do
+            var ipackages = [  #{escape_nested_list(@ipackages)}];
+            renderPackagesProjectsTable({packages: ipackages, length: 10, name: 'ipackages_wrapper'});
+      - else
+        %p
+          %i
+            = @displayed_user.login
+            is not involved in any packages
+    #iprojects.tab
+      - if @iprojects.blank?
+        %p
+          %i
+            = @displayed_user.login
+            is not involved in any projects
+      - else
+        #projects_table_wrapper{"data-url" => "#{url_for(controller: 'project', action: 'show', project: 'REPLACEIT')}"}
+          - content_for :head_javascript do
+            var main_projects = [ #{escape_nested_list(@iprojects)} ];
+            var excl_projects = [];
+          - content_for :ready_function do
+            renderProjectsTable(length=10);
+            $('#excludefilter').change(renderProjectsTable);
+    #iowned.tab
+      - if @owned.blank?
+        %p
+          %i
+            = @displayed_user.login
+            is not involved in any projects
+      - else
+        #iowned_wrapper{ "data-url": url_for(controller: 'package', action: 'show', project: 'REPLACEPRJ', package: 'REPLACEPKG') }
+          - content_for :ready_function do
+            var iowned = [ #{escape_nested_list(@owned)} ];
+            renderPackagesProjectsTable({packages: iowned, length: 15, name: 'iowned_wrapper'});
+.clear
+- if @is_displayed_user
+  .grid_16.alpha.omega.box.box-shadow
+    #reviews
+      .box-header.header-tabs
+        %ul
+          %li
+            %a{href: "#reviews_in", title: "Requests that #{@displayed_user.login} has to review"} Incoming Reviews
+            = sprite_tag('reload', title: 'Reload', class: 'result_reload', data: { table: 'reviews_in_table' })
+            = image_tag('ajax-loader.gif', class: 'result_spinner hidden')
+      #reviews_in.tab
```

Comment:
Class should be before id: https://github.com/brigade/haml-lint/blob/master/lib/haml_lint/linter/README.md#classesbeforeids